src/main/flight/pid.c

```diff
@@ -129,13 +129,19 @@ static FAST_RAM_ZERO_INIT float airmodeThrottleOffsetLimit;
 
 PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, PID_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 9);
 
+#ifdef USE_INTEGRATED_YAW_CONTROL
+#define YAW_PID_DEFAULTS [PID_YAW] =   { 30, 10, 10, 80 }
+#else
+#define YAW_PID_DEFAULTS [PID_YAW] =   { 45, 100, 0, 100 }
```

Comment:
The fact that the defaults are used both here and in `config.c` makes this prone to being broken when defaults are changed in the future.

I think what we should be doing is define the defaults separately here:

```
#define YAW_PID_DEFAULT_P 45
#define YAW_PID_DEFAULT_I 100
#define YAW_PID_DEFAULT_D 0
#define YAW_PID_DEFAULT_F 100
```

and then use them here

```
[PID_YAW] = { YAW_PID_DEFAULT_P, YAW_PID_DEFAULT_I, YAW_PID_DEFAULT_D, YAW_PID_DEFAULT_F }
```

and in `config.c`:

```
mutableProfile->pid[PID_YAW].P = YAW_PID_DEFAULT_P;
[...]
```

Reply:
Yeah, you're so right. And my lazy self knew it when writing these lines, too. Thanks for reigning it in!

Reply:
:-) Much better now.